specification/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,39 @@
+# Commitment Discount Type
+
+A commitment-based discount is a contractual commitment in exchange for a discounted unit price. Commitment discounts are available as spend-based or resource-based discounts, depending on the CSP offering them and the particular service.
+
+This column determines whether the commitment is based on a minimum spend or minimum usage.
+
+The CommitmentDiscountType column MUST be present in the billing data. This column must be of type String and MUST NOT contain null values when a charge IS related to a commitment-based discount. When a charge IS NOT associated with a commitment-based discount, the column MUST be null. The CommitmentDiscountType MUST be one of the allowed values.
+
+## Column ID
+
+CommitmentDiscountType
+
+## Display name
+
+Commitment Discount Type
+
+## Description
+
+The type of commitment agreed to in exchange for the discounted unit price.
+
+## Content constraints
+
+|    Constraint   |      Value       |
+|:----------------|:-----------------|
+| Column required | True             |
+| Data type       | String           |
+| Allows nulls    | True             |
+| Value format    | List of values   |
+
+Allowed values:
+
+| Value      | Description                                                                                                                                                                   |
+|:---------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Spend-based    | Commitments are purchased and measured in terms of the dollars per hour of equivalent on-demand spend.                                                                    |
+| Resource-based | Commitments are purchased and measured in terms of the underlying resources.                                                                                              |
```

Comment:
Technically, both quantity and cost are based on resources. It's probably more accurate to refer to this as either a "Quantity-Based" or "Usage-Based". 

Just to throw out another idea: We could use the column name that is committed. I'm not sure that's ideal, but just wanted to share in case that sparks something.
```suggestion
| Usage-Based | Commitments are purchased and measured in terms of the underlying resource usage.                                                                                              |
```

Reply:
Is this something I should put in a poll to a wider audience to get a quorum?

Reply:
That might be quickest. If we feel we have agreement here, then there's no need, but polls do tend to get relatively quick closure.

Reply:
I am now leaning more towards Usage-Based after reading through some of the other polls. I'll still put it to a poll to see if there's something we have missed if we change to this

Reply:
Actually, no, I think we agree on this - no one else has commented, so I'll switch to Usage-Based and anyone can comment here if they feel strongly about it.